src/server/v2.0/handler/preheat.go

```diff
@@ -248,6 +251,59 @@ func (api *preheatAPI) ListPolicies(ctx context.Context, params operation.ListPo
 		WithLink(api.Links(ctx, params.HTTPRequest.URL, total, query.PageNumber, query.PageSize).String())
 }
 
+// ManualPreheat is manual preheat
+func (api *preheatAPI) ManualPreheat(ctx context.Context, params operation.ManualPreheatParams) middleware.Responder {
+	project, err := api.projectCtl.GetByName(ctx, params.ProjectName)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	policy, err := api.preheatCtl.GetPolicyByName(ctx, project.ProjectID, params.PreheatPolicyName)
+	if err != nil {
+		return api.SendError(ctx, err)
+	}
+
+	_, err = api.enforcer.EnforcePolicy(ctx, policy.ID)
```

Comment:
I suggest accepting the returned execution id and return it to the front end with a proper format:

  - Option1: JSON data
  - Option2: add a location header in the response

Reply:
Great, I'll return JOSN data include ID.